### DISCUSSION TITLE
Wired XBOX ONE controller not Working

### DISCUSSION THREAD

**Discussion by dilankenner** [cathery/sys-con#51] [Troubleshooting]
The controller is recognized, shows up icon, but none of the inputs works.
Model: DEX XC-02 (WIRED CONTROLLER FOR XB ONE )
![screenshot](https://user-images.githubusercontent.com/42393491/68544602-b2a60a00-03a3-11ea-8391-ac19859bedf8.jpg)
[log.txt](https://github.com/cathery/sys-con/files/3828483/log.txt)

**Comment by cathery** [comment]
It's currently being investigated. Have you installed any firmware updates to the controller recently? Can you download an earlier version [0.2.2](https://github.com/cathery/sys-con/releases/tag/v0.2.2) and tell me if it works?

**Comment by dilankenner** [comment]
I tried to update, but it didn't work. Maybe it's not original. :/

**Comment by cathery** [comment]
Alright, what about 0.2.2?

**Comment by dilankenner** [comment]
Same thing. Shows up icon, but none of the inputs works.

**Comment by cathery** [comment]
Someone has reported that this build works for them. Can you try it? https://files.catbox.moe/1wse26.rar

**Comment by dilankenner** [comment]
I'll give it a try and report here as soon as I can.

**Comment by dilankenner** [comment]
Nope, same thing.

**Comment by cathery** [comment]
What about this one https://files.catbox.moe/jwliyq.rar

**Comment by dilankenner** [comment]
nothing yet. The lights keep blinking, the controller icon shows up, but nothing works. I'm testing directly on the tablet, do you want me to test on the dock?

**Comment by cathery** [comment]
Yea. Otherwise you need a cable that supports OTG undocked.

**Comment by dilankenner** [comment]
Yes, I'm using an OTG cable

**Comment by cathery** [comment]
Well whatever works docked also works undocked, and if one of them works but the other one doesn't then it's an issue with the cable/controller. So let me know if your controller works docked on 0.4.2

**Comment by ghost** [comment]
i would suggest to add NACOON Controller compability ;D

**Comment by dilankenner** [comment]
> i would suggest to add NACOON Controller compability ;D

How do i do that?

**Comment by HylianEternal** [comment]
Just to chime in on this real quick, I'm experiencing the same issue with my Xbox Elite Series 2 Controller plugged in via USB. I've tried 0.4.2, 0.3.0 and 0.2.3.1 and on each version the console detects the controller, but doesn't appear to be receiving or accepting any inputs. If there's anything I can do to help you with this such as testing a build please let me know :)

**Comment by cathery** [comment]
Oh that's weird, I've released 0.4.2 specifically to fix elite controllers, let me know if this version works https://files.catbox.moe/jwliyq.rar

**Comment by HylianEternal** [comment]
Unfortunately I'm still getting the same result as before. Is there a way I can send you a log of some sort?

**Comment by cathery** [comment]
Yeah, it's located at `sdmc:/config/sys-con/log.txt`

**Comment by HylianEternal** [comment]
[log.txt](https://github.com/cathery/sys-con/files/3837823/log.txt)
Here you go, hopefully this helps. Let me know if there's anything else I can do to help.

**Comment by HylianEternal** [comment]
Something I just realized is that I updated my controller's firmware recently, could that have something to do with the issue?

**Comment by cathery** [comment]
I have no idea. Elite series 2 released like what, 2 weeks ago? It's a safe bet your controller is not supported entirely.

**Comment by HylianEternal** [comment]
That's what I'm thinking too. Hopefully support can be added at some point, if you need me to test anything just let me know.

**Comment by cathery** [comment]
@dilankenner I just got myself a USB-C OTG adapter and all the official controllers I have work flawlessly. The only one that didn't work was a chinese DS3 fake. So perhaps you have the same issue? A knock-off brand that doesn't support OTG?

**Comment by dilankenner** [comment]
I will try it again using the dock and the most recent version of sys-con and see if it works.

**Comment by dilankenner** [comment]
Well, I think the only way for me is to buy another controller.
